### PR TITLE
u8g2: add support for menus

### PR DESCRIPTION
**Pull request by bradjc**
This also adds a simple test app where the interface can control LEDs.

### CODE REVIEW

**Comment by bradjc** [comment]
C is not a serious language. This issue appears to be ordering with the header files included.

Trying to fix here: https://github.com/olikraus/u8g2/pull/2414

File: examples/tests/u8g2-demos/led-menu/main.c [hunk 1]
```diff
@@ -0,0 +1,221 @@
+#include <screen.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <button.h>
+#include <led.h>
+#include <screen.h>
+#include <timer.h>
+
+// These have to be included before mui.h
```
Comment (ppannuto) [inline examples/tests/u8g2-demos/led-menu/main.c:10]: Is this still true after your PR got merged on the other repo?

File: examples/tests/u8g2-demos/led-menu/main.c [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    case MUIF_MSG_DRAW:
+      u8g2_DrawHLine(&u8g2, 0, mui_get_y(mui), u8g2_GetDisplayWidth(&u8g2));
+      break;
+  }
+  return 0;
+}
+
+
+muif_t muif_list[] = {
+  // // normal text style
```
Comment (ppannuto) [inline examples/tests/u8g2-demos/led-menu/main.c:51]:
```suggestion
  // normal text style
```

**Review by ppannuto — DISMISSED** [review]
Two relatively unimportant cosmetic questions — merge at your discretion

File: examples/tests/u8g2-demos/led-menu/main.c (hunk 1 above)
Comment (bradjc) [reply examples/tests/u8g2-demos/led-menu/main.c:10]: Probably not, but I didn't update the hash.

**Review by ppannuto — APPROVED** [review]

**Pull request merged**